src/operator/mergeScan.ts

```diff
@@ -9,28 +9,45 @@ import { OuterSubscriber } from '../OuterSubscriber';
 import { InnerSubscriber } from '../InnerSubscriber';
 
 /**
- * @param project
- * @param seed
- * @param concurrent
- * @return {Observable<R>|WebSocketSubject<T>|Observable<T>}
+ * Applies an accumulator function over the source Observable where the
+ * accumulator function itself returns an Observable, then each intermediate
+ * Observable returned is merged into the output Observable.
+ *
+ * <span class="informal">It's like {@link scan}, but the Observables returned
+ * by the accumulator are merged into the outer Observable.</span>
+ *
+ * @example <caption>Count the number of click events</caption>
```

Comment:
Open for suggestions for a less contrived example. Here's a thought:

```javascript
const clicks = Observable.fromEvent(
  document.querySelector('button'),
  'click'
);

clicks
  .map((_, i) => i)
  .mergeScan((acc, attempts) => {
    return Observable
      .timer(BACKOFF_MULTIPLIER * attempts)
      .mapTo(attempts + 1);
  }, 0)
  .subscribe(v => {
    console.log(v);
  });
```
http://jsbin.com/dumanuyuxa/1/edit?js,console,output

Still a bit contrived without good context...